packages/sdk/src/types/Rewards.ts

```diff
@@ -137,8 +140,6 @@ export interface RewardRedemptionParams {
 	id?: string
 }
 
-// domain types
```

Comment:
why you have deleted this ? under this line types are new. Domain types, not legacy

Reply:
why?